plugin/builtin/patchtransformer/PatchTransformer.go

```
@@ -100,6 +100,11 @@ func (p *plugin) transformStrategicMerge(m resmap.ResMap, patch *resource.Resour
 	if err != nil {
 		return err
 	}
+
+	if p.Options["allowNoTargetMatch"] {
+		log.Println("Warning: patches target not found for Target")
+	}
```

We still need to return an error if `allowNoTargetMatch` is not set?